odc/stats/_text.py

```diff
@@ -81,6 +81,22 @@ def parse_yaml_file_or_inline(s: str) -> Dict[str, Any]:
     return result
 
 
+def load_yaml_remote(yaml_url: str) -> Dict[str, Any]:
+    """
+       Open a yaml file remotely and return the parsed yaml document 
+    """
+    import urllib
```

Comment:
This function has many problems. 
1. It would need to import `urllib.error` to access the `urllib.error.URLError` exception class.
2. However, testing now, `fsspec` raises a `FileNotFoundError` if it can't access a file, and `yaml` raises `yaml.scanner.ScannerError` if it can't parse the input, so there's no point trying to catch `URLError`.
3. There is no `{value}` in scope for the `f-string` on line 95.

I think it would be reasonable to skip the Exception handling entirely here, since there's no good way to resolve the error.

OR

Alternatively, we could catch the exceptions and log a message saying `f'unable to load {yaml_url}` and then have a bare `raise` to re-raise the current exception.

OR

Define a new  `class YamlLoadError(Exception)`, and `raise YamlLoadError(f'Unable to load {yaml_url}')`  from inside the `except`.